test/clientAuth-bearerBody-test.js

```diff
@@ -0,0 +1,101 @@
+const assert = require('chai').assert;
+const AccessTokenRequest =
+    require('../oicMsg/oauth2/init.js').AccessTokenRequest;
+const AccessTokenResponse =
+    require('../oicMsg/oauth2/init.js').AccessTokenResponse;
+const AuthorizationRequest =
+    require('../oicMsg/oauth2/init.js').AuthorizationRequest;
+const AuthorizationResponse =
+    require('../oicMsg/oauth2/init.js').AuthorizationResponse;
+const BearerBody = require('../src/clientAuth/bearerBody').BearerBody;
+const Client = require('../src/oauth2/init').Client;
+const CLIENT_AUTHN_METHOD =
+    require('../src/clientAuth/clientAuth').CLIENT_AUTHN_METHOD;
+const ResourceRequest = require('../oicMsg/oauth2/init.js').ResourceRequest;
+
+const CLIENT_ID = 'A';
+
+const CLIENT_CONF = {
+  'issuer': 'https://example.com/as',
+  'redirect_uris': ['https://example.com/cli/authz_cb'],
+  'client_secret': 'boarding pass',
+  'client_id': CLIENT_ID
+};
+
+const REQ_ARGS = {
+  'redirect_uri': 'https://example.com/rp/cb',
+  'response_type': 'code'
+};
+
+function getClient() {
+  let client = new Client();
+  client.init(CLIENT_AUTHN_METHOD, CLIENT_CONF);
+  let sdb = client.clientInfo.stateDb;
+  sdb.dict = {};
+  sdb.dict['ABCDE'] = {'code': 'accessCode'};
+  client.clientInfo.clientSecret = 'boardingPass';
+  return client;
+}
+
+describe('Test bearer body', function() {
```

Comment:
Use arrow functions `() => {}` for anonymous functions (here and everywhere else below).

Note: you can declare functions the way you did above though for `getClient`.

Reply:
Fixed